### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

File: calibration/FLORDG/CGINS-FLORDG-03304__20220914.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3304,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_3304_Calibration_Files_2022-09-14.zip > FLBBSB-3304 Characterization Sheet.pdf
+3304,CC_dark_counts_chlorophyll_a,0.066,in Volts (because analog signal going to CTDBP)
+3304,CC_dark_counts_volume_scatter,0.0880,in Volts (because analog signal going to CTDBP)
+3304,CC_depolarization_ratio,0.039,Constant
+3304,CC_measurement_wavelength,700,Constant
+3304,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+3304,CC_scale_factor_volume_scatter,1.342E-03,in Volts (because analog signal going to CTDBP)
+3304,CC_scattering_angle,140,Constant
```
Comment (batryn) [inline calibration/FLORDG/CGINS-FLORDG-03304__20220914.csv:9]: Looks good

File: calibration/FLORDG/CGINS-FLORDG-03382__20220914.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3382,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_3382_Calibration_Files_2022-09-14.zip > FLBBSB-3382 Characterization Sheet.pdf
+3382,CC_dark_counts_chlorophyll_a,0.079,in Volts (because analog signal going to CTDBP)
+3382,CC_dark_counts_volume_scatter,0.0640,in Volts (because analog signal going to CTDBP)
+3382,CC_depolarization_ratio,0.039,Constant
+3382,CC_measurement_wavelength,700,Constant
+3382,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+3382,CC_scale_factor_volume_scatter,1.338E-03,in Volts (because analog signal going to CTDBP)
+3382,CC_scattering_angle,140,Constant
```
Comment (batryn) [inline calibration/FLORDG/CGINS-FLORDG-03382__20220914.csv:9]: Looks good

File: calibration/FLORDG/CGINS-FLORDG-04030__20220915.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4030,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_4030_Calibration_Files_2022-09-15.zip > FLBBSB-4030 Characterization Sheet.pdf
+4030,CC_dark_counts_chlorophyll_a,0.054,in Volts (because analog signal going to CTDBP)
+4030,CC_dark_counts_volume_scatter,0.0760,
+4030,CC_depolarization_ratio,0.039,Constant
+4030,CC_measurement_wavelength,700,Constant
+4030,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+4030,CC_scale_factor_volume_scatter,1.324E-03,in Volts (because analog signal going to CTDBP)
+4030,CC_scattering_angle,140,Constant
```
Comment (batryn) [inline calibration/FLORDG/CGINS-FLORDG-04030__20220915.csv:9]: Looks good

File: calibration/FLORDG/CGINS-FLORDG-04031__20220914.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4031,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_4031_Calibration_Files_2022-09-14.zip > FLBBSB-4031 Characterization Sheet.pdf
+4031,CC_dark_counts_chlorophyll_a,0.047,in Volts (because analog signal going to CTDBP)
+4031,CC_dark_counts_volume_scatter,0.0690,
+4031,CC_depolarization_ratio,0.039,Constant
+4031,CC_measurement_wavelength,700,Constant
+4031,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+4031,CC_scale_factor_volume_scatter,1.231E-03,in Volts (because analog signal going to CTDBP)
+4031,CC_scattering_angle,140,Constant
```
Comment (batryn) [inline calibration/FLORDG/CGINS-FLORDG-04031__20220914.csv:9]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

File: calibration/FLORDG/CGINS-FLORDG-03304__20220914.csv (hunk 1 above)
Comment (asmith-whoi) [inline calibration/FLORDG/CGINS-FLORDG-03304__20220914.csv:9]: OK

File: calibration/FLORDG/CGINS-FLORDG-03382__20220914.csv (hunk 2 above)
Comment (asmith-whoi) [inline calibration/FLORDG/CGINS-FLORDG-03382__20220914.csv:9]: OK

File: calibration/FLORDG/CGINS-FLORDG-04030__20220915.csv (hunk 3 above)
Comment (asmith-whoi) [inline calibration/FLORDG/CGINS-FLORDG-04030__20220915.csv:9]: OK

File: calibration/FLORDG/CGINS-FLORDG-04031__20220914.csv (hunk 4 above)
Comment (asmith-whoi) [inline calibration/FLORDG/CGINS-FLORDG-04031__20220914.csv:9]: OK

**Review by asmith-whoi — APPROVED** [review]

**Pull request merged**